travis/apis.py

```diff
@@ -0,0 +1,137 @@
+# coding: utf-8
+
+import os
+import tempfile
+import time
+import requests
+import json
+
+from travis_helpers import red, yellow, yellow_light
+
+
+class Request(object):
+
+    def __init__(self):
+        self.session = requests.Session()
+
+    def _check(self):
+        if not self._token:
+            print(yellow_light("WARNING! Token not defined- "
+                               "exiting early."))
+            exit(1)
```

Comment:
@moylop260 
Done, I created the method _check and this method is called into update method